src/visual/Slider.js

```diff
@@ -1194,6 +1218,7 @@ export class Slider extends util.mix(VisualStim).with(ColorMixin, WindowMixin)
 			labelText.rotation = -(this._ori + this._labelOri) * Math.PI / 180;
 			labelText.anchor = this._labelAnchor;
 			labelText.alpha = 1;
+			this._pixiLabels[l] = labelText;
```

Comment:
This is functionally fine but why not replace `const labelText =` by `this._pixiLabels[l] =` from the start?